recipes/xerces-c/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: xerces-c
+  version: 3.1.3
+
+source:
+  url: http://mirrors.sonic.net/apache//xerces/c/3/sources/xerces-c-3.1.3.tar.gz # [not win]
+  fn: xerces-c-3.1.3.tar.gz # [not win]
+  md5: 70320ab0e3269e47d978a6ca0c0e1e2d # [not win]
+
+  url: http://mirrors.sonic.net/apache//xerces/c/3/sources/xerces-c-3.1.3.zip # [win]
+  fn: xerces-c-3.1.3.zip # [win]
+  md5: 4b3ab521a3854c774b267005fb4d8ebd # [win]
+
+build:
+  features:
+    - vc9    # [win and py27]
+    - vc10   # [win and py34]
+    - vc14   # [win and py35]
+
```

Comment:
Again missing test section.

How about `- exit $(test -f ${PREFIX}/lib/libxerces-c.a)  # [not win]`